src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
+	const path = 'communications';
+	const { urlname, conversationId } = params;
+
+	let endpoint;
+	switch (params.method) {
+		case 'getAll':
+			endpoint = `${urlname}/${path}`;
```

Comment:
when using `switch`, don't forget to put on the `break`s. This `switch` will currently _always_ return ``endpoint = `${urlname}/${path}/`;``